tests/components/plum_lightpad/test_config_flow.py

```diff
@@ -0,0 +1,117 @@
+"""Test the Plum Lightpad config flow."""
+from requests.exceptions import ConnectTimeout
+
+from homeassistant import config_entries, setup
+from homeassistant.components.plum_lightpad.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)